internal/servicedeployer/compose.go

```diff
@@ -75,7 +75,7 @@ func (d *DockerComposeServiceDeployer) SetUp(ctx context.Context, svcInfo Servic
 	logger.Debug("setting up service using Docker Compose service deployer")
 	service := dockerComposeDeployedService{
 		ymlPaths: d.ymlPaths,
-		project:  "elastic-package-service",
+		project:  fmt.Sprintf("elastic-package-service-%s", svcInfo.Test.RunID),
```

Comment:
This change changes also the compose project name used without independent Elastic Agents.
This compose project name should not be taken into account in any package.

Should we keep the same compose project name in case independent Elastic Agents are not used (`d.deployIndependentAgent`)?

Reply:
It looks good like this.